js/src/main/scala/ru/org/codingteam/euclid/js/io/DisplayWrapper.scala

```diff
@@ -0,0 +1,24 @@
+package ru.org.codingteam.euclid.js.io
+
+import ru.org.codingteam.euclid.io.{Display, TextBlockSize, TextMeasurementService}
+import ru.org.codingteam.rotjs.ROT
+
+class DisplayWrapper(display: ROT.Display) extends Display {
```

Comment:
`DisplayWrapper` is a really vague name. This is literally an adapter of `ROT.Display` to our custom `Display` interface (if I understood correctly). Why not call that adapter `RotDisplay` or something like that?

Also, since we adapting incompatible stuff to our custom unified interface, maybe it's worth making that interface a typeclass? :) Not calling for an action here or anything. Just saying my thoughts out loud.

Reply:
Yes, `RotDisplay` is a good name, I'll change it. Regarding the typeclass — well, I'm honestly not sure if we actually  want to make the typeclass here. I can't see what benefit could it give.